src/seq_mv/csr_matvec_oomp.c

```diff
@@ -34,7 +34,7 @@ hypre_CSRMatrixMatvecOMPOffload( HYPRE_Int        trans,
 
    if (trans)
    {
-      hypre_CSRMatrixTransposeDevice(A, &B, 1);
+      hypre_CSRMatrixTranspose(A, &B, 1);
```

Comment:
This will lead to use `hypre_CSRMatrixTransposeHost` I guess, which is fine since `hypre_CSRMatrixTransposeDevice` doesn't work with OMPOffload (no-op) either. We don't currently support this anyways.